nipy/algorithms/registration/tests/test_fmri_realign4d.py

```diff
@@ -23,6 +25,9 @@
 from ..affine import Rigid
 
 im = load_image(funcfile)
```

Comment:
I'll get on the rest, but I think it is best if we do this in a separate commit.